python/adelphi/adelphi/store.py

```diff
@@ -0,0 +1,61 @@
+# Functions to facilitate interactions with the underlying data store
```

Comment:
Not using the name "cassandra" for this module in order to avoid any interference with the Cassandra driver